editors/code/package.json

```diff
@@ -6,7 +6,7 @@
     "private": true,
     "icon": "icon.png",
     "//": "The real version is in release.yaml, this one just needs to be bigger",
-    "version": "0.2.0-dev",
+    "version": "0.2.20200211-dev",
```

Comment:
This won't be changed every release, it's just that when running the extension in the debugger in prebuilt binaries mode the version must look like one in `package.json` which is deployed to the marketplace, otherwise it crashes on the regular expression [here](https://github.com/rust-analyzer/rust-analyzer/blob/325eba58a286c147f19dada5f205aa9e2ec6f391/editors/code/src/config.ts#L34). Though I won't call `package.json` a source code, it is a common practice to bump the version in `package.json` upon each release. If you don't want bumping the version, leaving it hardcoded on `20200211` won't harm.